application/src/main/java/org/opentripplanner/graph_builder/GraphBuilderDataSources.java

```diff
@@ -108,26 +108,26 @@ public boolean has(FileType type) {
     return inputData.containsKey(type);
   }
 
-  public Iterable<ConfiguredDataSource<OsmExtractParameters>> getOsmConfiguredDatasource() {
+  public Iterable<ConfiguredDataSource<OsmExtractParameters>> getOsmConfiguredDataSource() {
```

Comment:
If you already touch all the method names, you could also bring them in line with the convention and drop the `get`.